### DISCUSSION TITLE
Delete Button?

### DISCUSSION THREAD

**Discussion by jkrauska** [tphakala/birdnet-go#338] [Ideas]
When moving a mic or if I know a recording is wrong/bad, I'd like to intentionally delete it.  Can we add a trashcan icon in the UI (maybe in an admin mode) to intentionally delete a clip?

**Comment by Tserppa** [comment]
I would like this feature as well. It would be a simple solution to keep the data valid.

**Comment by tphakala** [comment]
Yeah UI definetely needs this, I am working on getting species exclude/include/custom threshold ui in settings, should be ready around this weekend. Once it is in I will start working on this.

**Reply by tphakala** [reply]
This now implemented in [Nightly Build 20250210](https://github.com/tphakala/birdnet-go/releases/tag/nightly-20250210)

![image](https://github.com/user-attachments/assets/21b79a49-133a-4a6e-9283-e4721783e7c2)
